docs/user_guide.md

```diff
@@ -306,6 +308,36 @@ ucc_coll_score_map.c:185  UCC  INFO       RocmManaged: {0..4095}:TL_UCP:10 {4K..
 [...] snip
 ```
 
+The next release of UCC add the `UCC_COLL_TRACE` environment variable:
```

Comment:
@Sergei-Lebedev should we be more specific here?

Reply:
yes, UCC 1.2.0 to be precise

Reply:
Thanks @Sergei-Lebedev I have update to

UCC 1.2.0 or newer supports the `UCC_COLL_TRACE` environment variable:

resolving this.